### PR TITLE
I18n and updated scraping

### PR DESCRIPTION
**Pull request by samlee405**

### CODE REVIEW

File: server/database_setup.py [hunk 1]
```diff
@@ -153,11 +176,11 @@
...
                 # If this item belongs in a set, query the set and add the relationship to the record
-                if record["set"]:
-                    set = record["set"]
+                if record["setID"]:
+                    set = record["setID"]
                     set_record = (
                         base.db_session.query(ModelSet)
-                        .filter(ModelSet.name == set)
+                        .filter(ModelSet.dofus_db_id == set)
                         .first()
```
Comment (lightyearsaway) [inline server/database_setup.py]: Not part of this PR, but you can use `.one()` or `.one_or_none()` for this

File: server/app/schema.py [hunk 2]
```diff
@@ -315,6 +335,13 @@ def resolve_items(self, info):
         query = Item.get_query(info)
         return query.all()
 
+    # not sure if we'll end up using this but could be useful
+    sets = graphene.NonNull(graphene.List(graphene.NonNull(Set)))
```
Comment (lightyearsaway) [inline server/app/schema.py]: We should be careful about loading and sending this much data, it can't be good for our server/costs. For development it's OK, but we definitely should turn it off or paginate it when we go to production.

File: server/app/schema.py [hunk 3]
```diff
@@ -55,11 +57,20 @@ class Meta:
...
+        model = ModelItemTranslation
+        interfaces = (GlobalNode,)
+
+
 class Item(SQLAlchemyObjectType):
     stats = graphene.NonNull(
         graphene.List(graphene.NonNull(ItemStats))  # Use list instead of connection
     )
     conditions = graphene.NonNull(graphene.List(graphene.NonNull(ItemCondtions)))
+    item_translations = graphene.NonNull(
```
Comment (lightyearsaway) [inline server/app/schema.py]: I'm not sure how you plan to use this, but we should be able to guess the user's language using Accept-Language in `info.context.headers`

File: server/app/migrations/versions/ba9a71425f36_.py [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    op.add_column('set', sa.Column('dofus_db_id', sa.String(), nullable=False))
+    op.drop_column('set', 'name')
+    op.add_column('set_bonus', sa.Column('alt_stat', sa.String(), nullable=True))
+    op.alter_column('set_bonus', 'max_value',
+               existing_type=sa.INTEGER(),
+               nullable=True)
+    op.alter_column('set_bonus', 'stat',
+               existing_type=postgresql.ENUM('VITALITY', 'AP', 'MP', 'INITIATIVE', 'PROSPECTING', 'RANGE', 'SUMMON', 'WISDOM', 'STRENGTH', 'INTELLIGENCE', 'CHANCE', 'AGILITY', 'AP_PARRY', 'AP_REDUCTION', 'MP_PARRY', 'MP_REDUCTION', 'CRITICAL', 'HEALS', 'LOCK', 'DODGE', 'PCT_FINAL_DAMAGE', 'POWER', 'CRITICAL_DAMAGE', 'NEUTRAL_DAMAGE', 'EARTH_DAMAGE', 'FIRE_DAMAGE', 'WATER_DAMAGE', 'AIR_DAMAGE', 'REFLECT', 'TRAP_DAMAGE', 'TRAP_POWER', 'PUSHBACK_DAMAGE', 'PCT_SPELL_DAMAGE', 'PCT_WEAPON_DAMAGE', 'PCT_RANGED_DAMAGE', 'PCT_MELEE_DAMAGE', 'NEUTRAL_RES', 'PCT_NEUTRAL_RES', 'EARTH_RES', 'PCT_EARTH_RES', 'FIRE_RES', 'PCT_FIRE_RES', 'WATER_RES', 'PCT_WATER_RES', 'AIR_RES', 'PCT_AIR_RES', 'CRITICAL_RES', 'PUSHBACK_RES', 'PCT_RANGED_RES', 'PCT_MELEE_RES', name='stat'),
```
Comment (lightyearsaway) [inline server/app/migrations/versions/ba9a71425f36_.py]: We'll need to update this with pods

File: server/app/migrations/versions/ba9a71425f36_.py [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+Create Date: 2020-03-04 17:17:31.776338
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'ba9a71425f36'
+down_revision = '138ae7ddca87'
```
Comment (lightyearsaway) [inline server/app/migrations/versions/ba9a71425f36_.py]: Migration 285b17... also has the same down_revision. Maybe you should rebase and re-make this migration? Otherwise we'll have to look into how to merge them.

File: server/app/schema.py (hunk 2 above)
Comment (samlee405) [reply server/app/schema.py]: Yea I imagine we would be paginating like we would for items. I included it to sort of open up the discussion as to whether or not browsing sets is something desirable that we want to include as a feature.

File: server/app/schema.py (hunk 3 above)
Comment (samlee405) [reply server/app/schema.py]: So you want to query for the specific translation and only return that?

File: server/app/schema.py (hunk 2 above)
Comment (lightyearsaway) [reply server/app/schema.py]: Yeah, I definitely think so, equipping an entire set at once is a very common action and one that's too hidden on Dofusbook I think

File: server/app/schema.py (hunk 3 above)
Comment (lightyearsaway) [reply server/app/schema.py]: Yeah, I think so, have you found use cases for querying for translations?
